samples/sensor/fxas21002/sample.yaml

```diff
@@ -1,8 +1,8 @@
 sample:
-  description: TBD
-  name: TBD
+  name: FXAS21002 Gyroscope Sensor Sample
 tests:
   test:
-    build_only: true
-    platform_whitelist: hexiwear_k64
+    harness: sensor
     tags: samples sensor
+    platform_whitelist: hexiwear_k64
```

Comment:
does the fxas21002 appear on any other boards?

Reply:
it can be on other boards and on daughter boards, but I left it whitelisted because of other issues, ultimate goal is to remove all the whitelisting. There are some issues still with device naming convention which makes it fail if I build it for other boards.